doc/list.rst

```diff
@@ -1346,6 +1348,23 @@ List of Available Rules
    Part of rule sets `@PER <./ruleSets/PER.rst>`_ `@PER-CS <./ruleSets/PER-CS.rst>`_ `@PER-CS1.0 <./ruleSets/PER-CS1.0.rst>`_ `@PER-CS2.0 <./ruleSets/PER-CS2.0.rst>`_ `@PSR12 <./ruleSets/PSR12.rst>`_ `@PhpCsFixer <./ruleSets/PhpCsFixer.rst>`_ `@Symfony <./ruleSets/Symfony.rst>`_
```

Comment:
Shall I update references to `new_with_braces` in existing rule sets?

Reply:
Yes, we should switch to new rules in the rule sets. You may want to look at #5709 where new fixer was introduced instead of previous one, with proxying etc.

Reply:
Thank you, @Wirone!